src/clients/scopes/ClientScopes.tsx

```diff
@@ -110,11 +118,27 @@ export const ClientScopes = ({
 
     const filter =
       searchType === "name" ? nameFilter(search) : typeFilter(searchTypeType);
-    return rows.filter(filter).slice(first, Number(first) + Number(max));
+    const firstNum = Number(first);
+    const page = rows
+      .filter(filter)
+      .slice(firstNum, Number(first) + Number(max));
```

Comment:
You already have `firstNum` here which has been converted to a number.
```suggestion
      .slice(firstNum, firstNum + Number(max));
```